src/olympia/constants/activity.py

```diff
@@ -826,6 +826,24 @@ class FORCE_ENABLE(_LOG):
     short = _('Force enabled')
 
 
+class LOG_IN(_LOG):
+    id = 169
+    # Note: clear_old_user_data cron would delete the IPLog when needed, so we
+    # can keep the activity log, it just won't have any IP associated with it.
+    keep = True
```

Comment:
what are we keeping the activity log for, after the IP address log has been deleted?

Reply:
I guess it's not always that useful, but IMHO it's still useful to know an action took place, and what it was, even if you don't have the IP, so you can reconstruct what they did.